src/bh_db_worker.erl

```diff
@@ -38,46 +48,75 @@ squery(Pool, Sql) ->
 equery(Pool, Stmt, Params) ->
     case do_checkout(Pool, ?POOL_CHECKOUT_RETRIES) of
         {ok, Reference, Conn} ->
-            Res = epgsql:equery(Conn, Stmt, Params),
-            dispcount:checkin(Pool, Reference, Conn),
-            Res;
+            case epgsql:parse(Conn, Stmt) of
```

Comment:
So there’s no epgsqla:equery that does this for you like epgsql:equery does?

Reply:
Yeah, you have to do it like this for the async functions